app/com/linkedin/drelephant/ElephantRunner.java

```diff
@@ -46,7 +46,7 @@
   private static final Logger logger = Logger.getLogger(ElephantRunner.class);
 
   private static final long WAIT_INTERVAL = 60 * 1000;      // Interval between fetches and retries
-  private static final int EXECUTOR_NUM = 3;                // The number of executor threads to analyse the jobs
+  private static final int EXECUTOR_NUM = 5;                // The number of executor threads to analyse the jobs
```

Comment:
Sounds good for now. We need to move to a full asynchronous model to avoid such fine tuning.